src/main/resources/application.yml

```
@@ -24,6 +24,19 @@ spring:
   config:
     activate:
       on-profile: dev
+  datasource:
+    driver-class-name: com.mysql.cj.jdbc.Driver
+    url: jdbc:mysql://dev-api.gist-petition.com:3306/dev_petitiondb?serverTimezone=Asia/Seoul
```

```suggestion
    url: jdbc:mysql://localhost:3306/dev_petitiondb?serverTimezone=Asia/Seoul
```
지금의 구현과 `localhost`를 이용한 구현에는 어떠한 차이점이 있을까요?

---

아 제 로컬에서 테스트 해볼려고 저렇게 했었습니다 
배포서버와 mysql이 모두 같은 곳에 있으니 이제는 localhost로 바꿔도 될 것 같습니다.